src/constants/actions.js

```diff
@@ -3,3 +3,8 @@ export const FETCH_ACTIONS = {
   success: 'FETCH_SUCCESS',
   error: 'FETCH_ERROR'
 };
+
+export const AUTH_ACTIONS = {
+  activate: 'ACTIVATE_AUTH',
+  remove: 'REMOVE_AUTH'
```

Comment:
I think it would be better to name the continents as login and logout, just for semantics.

what do you think?